api/v1/src/accounts/index.js

```diff
@@ -682,11 +743,241 @@ accounts.get(['/projects/:projectId/accounts:register', '/accounts:register'], a
+ *     description: Returns a redirect response
+ *     operationId: activateAccount
+ *     tags:
+ *       - accounts
+ *     parameters:
+ *     - in: header
+ *       name: x-gcp-project-id
+ *       type: string
+ *       required: true
+ *     - in: body
+ *       name: account
+ *       description: Request parameters for Account
+ *       schema:
+ *         $ref: '#/definitions/Account'
+ *     responses:
+ *       201:
+ *         description: Account
+ *         schema:
+ *           type: object
+ *           properties:
+ *             success:
+ *               type: boolean
+ *               description: Success of the request
+ *             code:
+ *               type: integer
+ *               description: HTTP status code
+ *             data:
+ *               type: object
+ *               items:
+ *                 $ref: '#/definitions/Account'
+ *       404:
+ *         description: Error
+ *         schema:
+ *           $ref: '#/definitions/Error'
+ *       500:
+ *         description: Error
+ *         schema:
+ *           $ref: '#/definitions/Error'
+ */
+/**
+ * @swagger
+ *
+ * /accounts:reset:
+ *   options:
+ *     summary: CORS support
+ *     description: Enable CORS by returning correct headers
+ *     operationId: optionsResetAccount
+ *     security: [] # no security for preflight requests
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: Default response for CORS method
+ *         headers:
+ *           Access-Control-Allow-Headers:
+ *             type: "string"
+ *           Access-Control-Allow-Methods:
+ *             type: "string"
+ *           Access-Control-Allow-Origin:
+ *             type: "string"
+ *   post:
+ *     summary: Resets a marketplace account based off request body
+ *     description: Returns a redirect response
+ *     operationId: resetAccount
+ *     tags:
+ *       - accounts
+ *     parameters:
+ *     - in: header
+ *       name: x-gcp-project-id
+ *       type: string
+ *       required: true
+ *     - in: body
+ *       name: account
+ *       description: Request parameters for Account
+ *       schema:
+ *         $ref: '#/definitions/Account'
+ *     responses:
+ *       201:
+ *         description: Account
+ *         schema:
+ *           type: object
+ *           properties:
+ *             success:
+ *               type: boolean
+ *               description: Success of the request
+ *             code:
+ *               type: integer
+ *               description: HTTP status code
+ *             data:
+ *               type: object
+ *               items:
+ *                 $ref: '#/definitions/Account'
+ *       404:
+ *         description: Error
+ *         schema:
+ *           $ref: '#/definitions/Error'
+ *       500:
+ *         description: Error
+ *         schema:
+ *           $ref: '#/definitions/Error'
+ */
+/**
+ * @swagger
+ *
+ * /accounts:syncMarketplace:
+ *   options:
```

Comment:
I don't understand yet why this options is needed, though without it API gateway denies the post.